ape_safe/accounts.py

```diff
@@ -208,7 +208,63 @@ def prepare_transaction(self, txn: TransactionAPI) -> TransactionAPI:
         # NOTE: Need to override `AccountAPI` behavior for balance checks
         return self.provider.prepare_transaction(txn)
 
-    def call(self, txn: TransactionAPI, **call_kwargs) -> ReceiptAPI:  # type: ignore[override]
+    def _impersonated_call(self, txn: TransactionAPI, **safe_tx_kwargs) -> ReceiptAPI:
+        safe_tx = self.create_safe_tx(txn, **safe_tx_kwargs)
+        safe_tx_exec_args = list(safe_tx._body_["message"].values())
+        signatures = []
+        # Bypass signature collection logic and attempt to submit by impersonation
+        # NOTE: Only works for fork and local networks
+        if self.confirmations_required > 1:
+            safe_tx_hash = self.contract.getTransactionHash(*safe_tx_exec_args)
+            for signer_address in self.signers[: self.confirmations_required - 1]:
```

Comment:
no need to sort later if the singers come pre-sorted.

Reply:
Originally I was thinking of ordering `signers` but one of the features eventually is custom order of signers via config, although maybe having `signers` be ordered and `local_signers` (which could be configurable) not be ordered would make sense